app/js/lib/simple_currency_formatter.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * This number formatter class is a primitive subset of the intl library ( https://www.npmjs.com/package/intl ),
+ * which is not available in all browsers and node versions.
+ */
+
+var parseGermanFloat = function ( amountStr ) {
+		return parseFloat( String( amountStr ).replace( ',', '.' ) );
+	},
+	GermanCurrencyFormatter = {
+		format: function ( amountStr ) {
+			var amount = this.parse( amountStr );
+			return String( amount.toFixed( 2 ) ).replace( '.', ',' ) + String.fromCharCode( 160 ) + '€';
+		},
+		parse: parseGermanFloat
+	},
+	EnglishCurrencyFormatter = {
+		format: function ( amountStr ) {
+			var amount = this.parse( amountStr );
+			return String( amount.toFixed( 2 ) ) + String.fromCharCode( 160 ) + '€';
```

Comment:
Your'e right. I'd think the best approach would be to emulate the locale behavior of the JS  `Intl` library as close as possible. For the English locale, it puts the euro sign in the front. I'll change this.